src/WebServer_JSON.ino

```diff
@@ -3,7 +3,75 @@
 #include "src/Globals/Plugins.h"
 #include "StringProviderTypes.h"
 
+// ********************************************************************************
+// Web Interface get CSV value from task
+// ********************************************************************************
+void handle_csvval()
+{
+  String htmlData(25, '\0'); // Reserve for error message
```

Comment:
I'm not sure what this constructor should do, but it looks like you're calling it to store the integer value 25 in base-0 notation (no idea what that should do).
It is not a reserve + init to an empty string.

Reply:
I'm sorry i have a problems with my VSCode with detecting using headers.
Is it not a std::string?
Because if so, then it should create a string with 25 chars which of these are '\0' (string terminator)

Reply:
Nope, `String` is an Arduino specific object.
We don't use std::string in our code here. Not that it is forbidden, but more that this may be confusing when using `String` and `string` in the same project.

Reply:
Oh, ok :)
I'm pretty fresh in ESPEasy, so i haven't know. but ok i fixed it ;)